.gitignore

```diff
@@ -28,6 +28,7 @@ dependency-reduced-pom.xml
 # Random
 *.log
 *.bak
+.DS_Store
```

Comment:
Try not to do something like this.

Instead go to gitignore.io, add OSX to the list of things to ignore, and append the results to the file. Maybe Windows and Linux too: https://www.gitignore.io/api/linux%2Cwindows%2Cosx

```

# Created by https://www.gitignore.io/api/linux,windows,osx

### Linux ###
*~

# temporary files which can be created if a process still has a handle open of a deleted file
.fuse_hidden*

# KDE directory preferences
.directory

# Linux trash folder which might appear on any partition or disk
.Trash-*


### Windows ###
# Windows image file caches
Thumbs.db
ehthumbs.db

# Folder config file
Desktop.ini

# Recycle Bin used on file shares
$RECYCLE.BIN/

# Windows Installer files
*.cab
*.msi
*.msm
*.msp

# Windows shortcuts
*.lnk


### OSX ###
.DS_Store
.AppleDouble
.LSOverride

# Icon must end with two \r
Icon


# Thumbnails
._*

# Files that might appear in the root of a volume
.DocumentRevisions-V100
.fseventsd
.Spotlight-V100
.TemporaryItems
.Trashes
.VolumeIcon.icns

# Directories potentially created on remote AFP share
.AppleDB
.AppleDesktop
Network Trash Folder
Temporary Items
.apdisk
```